app/javascript/components/view/PaginationTableView.js

```diff
@@ -7,14 +7,21 @@ import PropTypes from "prop-types";
 import {observer} from "mobx-react"
 import "../eui.css"
 import "../pagination.css"
-import ColorFilterView from "./ColorFilterView";
+// import ColorFilterView from "./ColorFilterView";
```

Comment:
I think this should be put back to the way it was.

Reply:
back